sdk/attestation/azure-security-attestation/README.md

```diff
@@ -1,39 +1,362 @@
+
+
+## Prerequisites
+
+- [Azure Subscription][azure_subscription]. Sign up for a [free trial](https://azure.microsoft.com/pricing/free-trial/) or use your [MSDN subscriber benefits](https://azure.microsoft.com/pricing/member-offers/msdn-benefits-details/).
+- An existing [Azure Attestation instance][azure_attestation]. If you need to create an attestation instance, you can use the [Azure Cloud Shell][azure_cloud_shell] to create one with this Azure CLI command. Replace `<your-resource-group-name>` and `<your-instance-name>` with your own, unique names:
+
+```bash
+az attestation create --resource-group <your-resource-group-name> --name <your-service instance name>
+```
+
+## Download & Install
+
+### Install Dependencies
+
+#### Windows
+
+On Windows, dependencies are managed by [vcpkg](https://github.com/microsoft/vcpkg). You can reference the [Quick Start](https://github.com/microsoft/vcpkg#quick-start-windows) to quickly set yourself up.
+After Vcpkg is initialized and bootstrapped, you can install the dependencies:
+```BatchFile
+vcpkg.exe install curl:x64-windows-static
+vcpkg.exe install openssl:x64-windows-static
+```
+
+#### POSIX Platforms
+
+You can use the package manager on different POSIX platforms to install the dependencies. The dependencies to be installed are:
+
+  - CMake 3.13.0 or higher.
+  - OpenSSL.
+  - libcurl.
+
+### Build from Source
+
+First, download the repository to your local folder:
+```BatchFile
+git clone https://github.com/Azure/azure-sdk-for-cpp.git
+```
+
+#### Windows
+
+##### Use CMake to generate the solution file
+
+In a new folder you created under the root directory:
+```BatchFile
+cmake .. -A x64 -DCMAKE_TOOLCHAIN_FILE=<YOUR_VCPKG_INSTALL_DIR>/scripts/buildsystems/vcpkg.cmake
+cmake --build .
+```
+
+The built library will be in `.\sdk\<ProjectDir>\<Configuration>\` respectively for Azure Core and Azure Attestation. e.g. `azure_core.lib` will be in `.\sdk\core\azure-core\Debug` for debug configuration.
+
+##### Use Visual Studio's Open by folder feature
+Open the root folder of the library with Visual Studio's Open folder feature.
+
+If Vcpkg is not globally integrated, then you need to open CMakeSettings.json and change the `Make toolchain file to be <YOUR_VCPKG_INSTALL_DIR>/scripts/buildsystems/vcpkg.cmake` and save.
+Then you can build Azure Storage libraries by selecting the target in Visual Studio, or simply build all.
+The libraries will be in `<ProjectRoot>\out\build\<Configuration>\sdk\<LibraryName>` respectively.
+
+#### POSIX Platforms
+
+You can run the following command in a new folder created under the downloaded code's root folder to build the code.
+
+```bash
+cmake .. -DCMAKE_BUILD_TYPE=Debug
+cmake --build .
+```
+Then you can consume the built library with the header files.
+make/ninja install is work in progress.
+
+
+### Via NuGet
+WIP
+TODO when ready.
+
```

Comment:
I think we can remove this section for now, unless we have a strong reason to use NuGet as an acquisition point, today.

Reply:
Again, stolen from KV.